tools/train_linemod_template.cpp

```diff
@@ -261,15 +261,25 @@ main (int argc, char** argv)
   float max_height = std::numeric_limits<float>::max ();
   parse_argument (argc, argv, "-max_height", max_height);
 
+  uint32_t unorganized_pcd_cnt = 0;
```

Comment:
```suggestion
  std::size_t unorganized+pcd+cnt = 0;
```

Might be better to instead use a bool to detect a single organized file since that's the end aim

Reply:
Isn't that the same? `all_of(x)` == `not(any_of(not_x))` as per boolean logic